scripts/build/utils.js

```diff
@@ -395,6 +384,15 @@ function rm(dest, opts) {
 }
 exports.rm = rm;
 
+class Deferred {

```

Comment:
Is this being used?

Reply:
By `Debouncer` below, yeah.

Reply:
This really should just be a thing in JS, like a `Promise.defer()` that returns a deconstructed promise object.